internal/internal_task_pollers.go

```diff
@@ -474,19 +474,21 @@ func (wtp *workflowTaskPoller) RespondTaskCompletedWithMetrics(
 ) (response *workflowservice.RespondWorkflowTaskCompletedResponse, err error) {
 	metricsHandler := wtp.metricsHandler.WithTags(metrics.WorkflowTags(task.WorkflowType.GetName()))
 	if taskErr != nil {
-		wtp.logger.Warn("Failed to process workflow task.",
-			tagWorkflowType, task.WorkflowType.GetName(),
-			tagWorkflowID, task.WorkflowExecution.GetWorkflowId(),
-			tagRunID, task.WorkflowExecution.GetRunId(),
-			tagAttempt, task.Attempt,
-			tagError, taskErr)
 		failWorkflowTask := wtp.errorToFailWorkflowTask(task.TaskToken, taskErr)
+		completedRequest = failWorkflowTask
+
 		failureReason := "WorkflowError"
 		if failWorkflowTask.Cause == enumspb.WORKFLOW_TASK_FAILED_CAUSE_NON_DETERMINISTIC_ERROR {
 			failureReason = "NonDeterminismError"
 		}
+
 		incrementWorkflowTaskFailureCounter(metricsHandler, failureReason)
-		completedRequest = failWorkflowTask
+		wtp.logger.Warn("Failed to process workflow task.",
```

Comment:
What is the goal of this change?

Reply:
old change that I forgot to revert, reverted.